chain/messagepool/selection.go

```diff
@@ -548,7 +810,13 @@ func (mc *msgChain) Trim(gasLimit int64, mp *MessagePool, baseFee types.BigInt,
 		mc.gasReward = new(big.Int).Sub(mc.gasReward, gasReward)
 		mc.gasLimit -= mc.msgs[i].Message.GasLimit
 		if mc.gasLimit > 0 {
+			bp := 1.0
+			if mc.effPerf != 0 {
+				bp = mc.effPerf / mc.gasPerf
```

Comment:
Uhh, missed that.
if `mc.gasPerf == 0` then bp is `Inf` so effPerf is `Inf`

Reply:
but then effPerf will be 0, so we are actually fine?

Reply:
undefined, good thing you spotted.